app/views/facility_accounts_reconciliation/_action_row.html.haml

```diff
@@ -1,27 +1,30 @@
 .row.table-actions.form-horizontal
   .span2.select_all_none= select_all_link
+  - if show_order_status
```

Comment:
I wonder if we use this "action_row" partial in other places, like Move up, that could be causing spec failures.  
```suggestion
  - if local_assigns[:show_order_status]
```